org-code-javabuilder/theater/src/main/java/org/code/theater/Theater.java

```diff
@@ -0,0 +1,202 @@
+package org.code.theater;
+
+public enum Instrument {
+   PIANO, BASS
+}
+
+public static class Theater {
+   /**
+    * Returns the width of the theater canvas. Right now this will always be
+    * 400 pixels.
+    */
+   public static double getWidth() {
+   }
+   
+   /**
+    * Returns the height of the theater canvas. Right now this will always be
+    * 400 pixels.
+    */
+   public static double getHeight() {
+   }
+   
+   /**
+    * Plays the array of samples provided.
+    * 
+    * @param sound an array of samples to play.
+    */
+   public static void playSound(double[] sound) {
```

Comment:
Do we want to consider having a Sound class similar to how we have an Image class?  I think this would be more consistent and give us a lot more flexibility going forward.

Reply:
We opted for direct use of `double[]` as the lessons that use sounds are very array-oriented and this type makes the lessons clearer. We may add a sound object in the future, depending on how our use cases outside of the course take shape.